ciprs_reader/reader/util.py

```diff
@@ -9,9 +9,15 @@
 }
 
 
+BINARY_MAP = {
+    ParserMode.V1: "pdftotext",
+    ParserMode.V2: "pdftotext-4.03",
```

Comment:
Would it be possible to summarize, at a high level, how v1 and v2 differ when used by ciprs-reader? I see several small tweaks to the regex throughout the parsers, which will impact both versions I believe, so what does using v2 buy us?

Reply:
V1 uses an old version of poppler, which is a fork of xpdfreader. It's missing the `-table` option and the `-layout` option has different output compared between v1 and v2.

Most of the regex changes are handling the differences in white space between V1 and V2. It's usually as simple as adding a "\s" to the regex to handle extra newlines and spaces.

To answer your main question, using V2 buys us a (seemingly) deterministic way of parsing offenses with multiple lines of text. With the `-table` option, only available in V2, the second line of the multiline text is always underneath the start of the offense description. With `-layout`, the second line of  the multiline text is sometimes above and sometimes below the offense description. It might be possible to use `-layout` and  be able to parse the second line of the offense description in the proper order, but it seemed like it would take too much effort.